### PR TITLE
Improve DAML SDK documentation

### PR DESCRIPTION
**Pull request by miklos-da**
Improved DAML SDK documentation, focusing on quickstart guide:
 - Corrected typos.
 - Corrected some broken links.
 - Made sample code formatting consistent (variable naming, comments).

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/daml/intro/5_Restrictions.rst [hunk 1]
```diff
@@ -59,7 +59,7 @@ For example, the simple Iou in :ref:`simple_iou` allowed the no-op where the ``o
   :start-after: -- TRANSFER_TEST_BEGIN
   :end-before: -- TRANSFER_TEST_END
 
-Similarly, you can write a ``Redeem`` choice, which allows the ``owner`` to redeem an ``Iou`` during business hours on weekdays. The choice doesn't do anything other than archiving the ``SimpleIou``. (This assumes that actual cash changes hands off-ledger.)
+Similarly, you can write a ``Redeem`` choice, which allows the ``owner`` to redeem an ``Iou`` during business hours on weekdays. The choice doesn't do anything other than archiving the ``SimpleIou``. (This assumes that actual cash changes are handled off-ledger.)
```
Comment (SamirTalwar) [inline docs/source/daml/intro/5_Restrictions.rst]: I think this one made more sense before; "changes hands" is the phrase (meaning that party A gave something to party B).

File: docs/source/daml/intro/daml/daml-intro-5/Restrictions.daml [hunk 2]
```diff
@@ -104,37 +104,37 @@ test_restrictions = scenario do
...
 
 -- TRANSFER_TEST_BEGIN
-  -- Alice can't transfer to herself
+  -- Alice can't transfer to herself...
   submitMustFail alice do
     exercise iou Transfer with
       newOwner = alice
 
-  -- but can transfer to bob
+  -- ...but can transfer to Bob.
```
Comment (SamirTalwar) [inline docs/source/daml/intro/daml/daml-intro-5/Restrictions.daml]:
```suggestion
  -- ... but can transfer to Bob.
```

File: docs/source/getting-started/quickstart/template-root/daml/Tests/Iou.daml [hunk 3]
```diff
@@ -42,20 +42,20 @@ iou_test = scenario do
   iouBobCid <- submit bob do
     exercise iouTransferBobCid IouTransfer_Accept
 
-  -- Alice transfers some Iou's to Bob
+  -- Alice transfers part of split IOU to Bob.
```
Comment (SamirTalwar) [inline docs/source/getting-started/quickstart/template-root/daml/Tests/Iou.daml]:
```suggestion
  -- Alice splits the IOU and transfers part of it to Bob.
```

File: docs/source/daml/intro/5_Restrictions.rst (hunk 1 above)
Comment (miklos-da) [reply docs/source/daml/intro/5_Restrictions.rst]: You're right -- changed back.

File: docs/source/concepts/ledger-model/ledger-integrity.rst [hunk 4]
```diff
@@ -410,7 +410,7 @@ agree to this obligation. In other words, the painter did not *authorize*
 the creation of the obligation.
 
 In a DA ledger, a party can **authorize** a subaction of a commit in
-one of the two ways:
+either of the following way:
```
Comment (SamirTalwar) [inline docs/source/concepts/ledger-model/ledger-integrity.rst]: "Ways" was correct.

```suggestion
either of the following ways:
```

Comment (miklos-da) [reply docs/source/concepts/ledger-model/ledger-integrity.rst]: Thank you for your input -- changed accordingly.

**Pull request merged**